client/plots/hierCluster.js

```diff
@@ -114,7 +115,10 @@ export class HierCluster extends Matrix {
 					values: [
 						{
 							sample: column.name,
-							dt: this.state.config.dataType,
+							dt:
+								this.state.config.dataType == TermTypes.GENE_EXPRESSION
```

Comment:
defaults to one dt is always no good here. can introduce a new mapping from TermTypes to dt
`dt: TermType2Dt(this.state.config.dataType)` 
in which TermType2Dt() throws on unknown type